The-Chonky-Panda/Atelier
# Test ratings and reviews

## Description

Added more tests to ratings and reviews section and moved those into a subfolder `/test`.
Added `--collectCoverage` to our npm script to collect test coverage metrics, though it currently looks weird.
Made `starBreakdown` and `productBreakdown` a little bit more dynamic though they are still placeholders.
Made a tiny style change to `write new review` modal so that it appears in the middle of the page. (maybe style can shared with related products comparison modal as well?)

## How to test

1. `npm test`
2. Click on the `add a review` button and verify that the modal pops up in the middle of the page

## Notes


## Issue tracking

https://trello.com/c/w2IjVzuh
